src/sentry/static/sentry/app/components/charts/miniBarChart.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+
+import theme from 'app/utils/theme';
+import {getFormattedDate} from 'app/utils/dates';
+
+import BarChart from './barChart';
+import {truncationFormatter} from './utils';
+
+type Marker = {
+  name: string;
+  value: string | number | Date;
+  color: string;
+  symbolSize?: number;
+};
+
+const defaultProps = {
+  colors: [theme.gray300, theme.purple300, theme.purple400],
+  emphasisColors: [theme.gray400, theme.purple400, theme.purple500],
+};
+
+type Props = React.ComponentProps<typeof BarChart> & {
+  /**
+   * A list of series to be rendered as markLine components on the chart
+   * This is often used to indicate start/end markers on the xAxis
+   */
+  markers?: Marker[];
+  /**
+   * Whether timestamps are should be shown in UTC or local timezone.
+   */
+  utc?: boolean;
+
+  /**
+   * Colors to use on the chart.
+   */
+  colors: string[];
+
+  /**
+   * Hover state colors to use on the chart.
+   */
+  emphasisColors: string[];
```

Comment:
Should we move `colors` and `emphasisColors` to own `DefaultProps` type? Right now we can pass whatever we want into `defaultProps` object and the compiler would not complain.

Reply:
Sorry I misread this. I've added better types for defaultProps.